### PR TITLE
Update Steve Kirk

### PR DESCRIPTION
**Pull request by aghassemlouei**
This pull request adds information to Steve Kirk's (Lockheed) original profile, introduced in #60, and renames the associated images for consistency.

### CODE REVIEW

**Comment by attritionorg** [comment]
LGTM

File: people/stevekirk.json [hunk 1]
```diff
@@ -2,47 +2,73 @@
...
+    },
+    {
+      "sitename": "Saber Martial Arts Foundation",
+      "siteurl": "https://sabermartialarts.life/our-force-ghost/"
     }
   ],
   "contributions": [
     {
       "title": "DEF CON NOC",
       "url": "https://www.defconnetworking.org/",
```
Comment (mauvehed) [inline people/stevekirk.json]: This website has moved to https://noc.defcon.org as of a little while ago

**Review by mauvehed — CHANGES_REQUESTED** [review]

File: people/stevekirk.json (hunk 1 above)
Comment (aghassemlouei) [reply people/stevekirk.json]: URI updated; apologies for not validating everything. Changes made and pushed upstream!

Comment (mauvehed) [reply people/stevekirk.json]: @aghassemlouei I realize now that's on us. It was an old URL that we added and I flagged you for fixing! My bad! But all of your contributions are massively appreciated!!!!

**Review by mauvehed — APPROVED** [review]

File: people/stevekirk.json (hunk 1 above)
Comment (aghassemlouei) [reply people/stevekirk.json]: No need to apologize; more than happy to go through all the profiles and ensure that they are up to date and accurate if that's what will help!

**Pull request merged**